### DISCUSSION TITLE
List of community projects that work with argo-cd

### DISCUSSION THREAD

**Discussion by bozerkins** [argoproj/argo-cd#9199] [Ideas]
Good day!

Currently it's a little time consuming to collect necessary "additional" tools for argo-cd setup and figure out how they work together.

It would be great to have a list of argo-cd and community projects somewhere in the argo-cd README.md or documentation. It would 
1) Help new users find useful components for their setup faster
2) Help contributors and new community projects get recognized faster

Many open-source projects do that, and it makes life so much easier.

Thanks!

**Comment by terrytangyuan** [comment]
Have you seen this list (specifically the ecosystem projects section)? https://github.com/terrytangyuan/awesome-argo

**Reply by bozerkins** [reply]
I haven't! It's a wonderful list, thank you for sharing <3

It would be nice to have a link to this list in argo-cd readme, somewhere in the community section.

**Reply by terrytangyuan** [reply]
It's actually already at the top of https://github.com/argoproj/argo-cd#blogs-and-presentations

Feel free to send a PR to link it to other places where you think could increase visibility.
